### ISSUE TITLE
 when there are more  options  and Put <AutocompleteInput> at the end of the form will display not well

### ISSUE THREAD

**Issue by MoonSeeing** [marmelab/react-admin#3940]
**What you were expecting:**
AutocompleteInput options display well  when there are too many options .

**What happened instead:**
options  popup in the top of input .and when input some to search and the result will far away from the input .really ugly!

**Steps to reproduce:**
see [https://codesandbox.io/s/simple-ucjul](https://codesandbox.io/s/simple-ucjul)
1.see comment create page 
2.select Number field bottom of the form and input 1 to search

**Related code:**
```
       <AutocompleteInput    source="number" choices={[
                { id: '0', name: '0' },
                 { id: '1', name: '1' },
                 { id: '11', name: '11' },
                 { id: '2', name: '2' },
                 { id: '3', name: '3' },
                 { id: '4', name: '4' },
                 { id: '5', name: '5' },
                 { id: '6', name: '6' },
                 { id: '7', name: '7' },
                 { id: '8', name: '8' },
                 { id: '9', name: '9' },
                    ]} />
```

**Other information:**
<img width="330" alt="WeChat823951989bad7078a83d09d8a44e0fc3" src="https://user-images.githubusercontent.com/19420622/68265460-11f3d980-0087-11ea-9946-99980aff0d8c.png">

**Environment**

* React-admin version:next

**Comment by Kmaschta** [comment]
Confirmed! Thanks for reporting

**Comment by fzaninotto** [comment]
Fixed by #3951
